src/app/Scenes/MyCollection/Screens/ArtworkForm/MyCollectionArtworkForm.tsx

```diff
@@ -94,6 +94,7 @@ export const MyCollectionArtworkForm: React.FC<MyCollectionArtworkFormProps> = (
 
   const [loading, setLoading] = useState<boolean>(false)
   const [isArtworkSaved, setIsArtworkSaved] = useState<boolean>(false)
+  const [displayText, setDisplayText] = useState<string>("")
```

Comment:
A bit longer but like this it's immediately clear what this state is for.

```suggestion
  const [savingArtworkModalDisplayText, setSavingArtworkModalDisplayText] = useState<string>("")
```